membership_file/admin.py

```diff
@@ -1,16 +1,22 @@
 
 from datetime import datetime
+import unicodedata
 
 from django.contrib import admin, messages
 from import_export.admin import ExportActionMixin
-from import_export.formats.base_formats import CSV
+from import_export.formats.base_formats import CSV, TSV, ODS, XLSX
 
 from .forms import AdminMemberForm
 from .models import Member, MemberLog, MemberLogField, Room, MemberYear, Membership
 from core.admin import DisableModificationsAdminMixin, URLLinkInlineAdminMixin
 from membership_file.export import MemberResource, MembersFinancialResource
 from utils.forms import RequestUserToFormModelAdminMixin
 
+class TSVUnicodeBOM(TSV):
+    def get_title(self):
+        return "tsv with Unicode BOM marker"
+    def export_data(self, *args, **kwargs):
+        return unicodedata.lookup('ZERO WIDTH NO-BREAK SPACE') + super().export_data(*args, **kwargs)
```

Comment:
Why does this character need to be added in front of the TSV file? Can you leave a comment to explain that?

Reply:
I added a docstring now. Is this enough?